tests/models/whisper/test_whisper.py

```diff
@@ -0,0 +1,24 @@
+from transformers import WhisperProcessor, WhisperForConditionalGeneration
+from datasets import load_dataset
+
+
+def test_whisper(record_property):
+    record_property("model_name", "Whisper")
+
+    # load model and processor
+    processor = WhisperProcessor.from_pretrained("openai/whisper-small")
+    model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-small")
+    model.config.forced_decoder_ids = None
+
+    # load dummy dataset and read audio files
+    ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
+    sample = ds[0]["audio"]
+    input_features = processor(sample["array"], sampling_rate=sample["sampling_rate"], return_tensors="pt").input_features 
+
+    # generate token ids
+    predicted_ids = model.generate(input_features)
```

Comment:
From my understanding, `.generate` does extra processing to the outputs compared to calling `model(input_features)`. Can you confirm the outputs between these two methods just on CPU? The torch_ttnn backend calls the compiled `model(input_features)`, so there might be some mismatching issues. I know `.generate` is a very popular way of using models, so that might be something we can try to support in the near future.